doc/nrf/ug_nrf5340.rst

```diff
@@ -569,6 +569,8 @@ To enable the simultaneous update of multiple images in the MCUboot, set the fol
 * :kconfig:option:`CONFIG_PCD_APP` - Enable commands exchange with the network core.
 * :kconfig:option:`CONFIG_UPDATEABLE_IMAGE_NUMBER` - Enable support for multiple update partitions by setting this option to ``2``.
 
+After that the feature will be selected by default thanks to :kconfig:option:`CONFIG_NRF53_MULTI_IMAGE_UPDATE`.
```

Comment:
```suggestion
* :kconfig:option:`CONFIG_NRF53_MULTI_IMAGE_UPDATE` - Select the simultaneous multi-image DFU feature.
```
Suggesting to move this option to the beginning of the list above.

Reply:
I should have meant that: After that the feature will be <b>automatically</b> selected by default thanks to ...
So there is no real need to enable CONFIG_NRF53_MULTI_IMAGE_UPDAT manually. I just wan't to mention the option name so it became visible to the user.

Reply:
Okay. How about something like this then: "After setting these options, the :kconfig:option:`CONFIG_NRF53_MULTI_IMAGE_UPDATE` option selects this feature automatically by default."

Reply:
applied